exercises/standard_library_types/iterators4.rs

```diff
@@ -10,11 +11,8 @@ pub fn factorial(num: u64) -> u64 {
     // For an extra challenge, don't use:
     // - recursion
     // Execute `rustlings hint iterators4` for hints.
-    let mut result: u64 = 1;
-    for value in 1..num+1{
-       result *= value 
-    }
-    result
+    (1..num+1).reduce(|a,b| a * b ).unwrap()
```

Comment:
unwrap is not recommended since the result of reduce could be NONE and unwrap could throw error then